ADR/Components/Aerodynamic_components/Airfoil.py

```diff
@@ -45,61 +43,57 @@ def get_Cm(self, alpha):
 
     def generate_upper_surface_coordinates(self):
 
-        self.n = (
-            self.airfoil_x_coords.size
-        )  # número de elementos do vetor de coordenadas horizontais
-        self.minpos = np.argmin(
-            self.airfoil_x_coords
-        )  # posição do menor valor no vetor de coordenadas horizontais, determina a posição do bordo de ataque
+        self.n = self.airfoil_x_coords.size
+        # número de elementos do vetor de coordenadas horizontais
+        self.minpos = np.argmin(self.airfoil_x_coords)
```

Comment:
Need a line break between comment and new variable. Like this:

```
variable1 = value
# comment about variable1

variable2 = value
# comment about variable2
```

This is valid for the rest of the code below.